### DISCUSSION TITLE
Are discounts available?

### DISCUSSION THREAD

**Discussion by dimstefanakis** [gatsbyjs/gatsby#32089] [Help]
It seems like I can find anything else I need, but I don't see any references to discounts

**Comment by sslotsky** [comment]
There is some discount data available from the admin API, but I don't know enough to say whether it supports your use case. 

https://shopify.dev/docs/admin-api/graphql/reference/common-objects/queryroot/index

If any of the connections that you find here expose the data that you need, we can look into adding it.

**Reply by shawngrona** [reply]
It would be super useful to be able to query for discounts, in particular automatic discounts. 

If you could get the discounts then you could match them up with individual products and surface the available automatic discount to the user.  

The way I do that now, is to change the price on the individual variants of the product during the sale.  This is probably not best practice when it comes to record keeping, but its what shopify mentions https://help.shopify.com/en/manual/discounts/sales
